tests/test_functions.py

```diff
@@ -45,6 +50,57 @@ def background_json(tmpdir):
         "data": {"filename": str(tmpdir / "filename.txt"), "value": "some-value"},
     }
 
+def _createTestEvent():
+    event = (
+        v1.Event()
+            .SetContentType("application/json")
+            .SetData('{"name":"john"}')
+            .SetEventID("my-id")
+            .SetSource("from-galaxy-far-far-away")
+            .SetEventTime("tomorrow")
+            .SetEventType("cloudevent.greet.you")
+    )
+    return event
```

Comment:
This should be a [pytest fixture](https://docs.pytest.org/en/stable/fixture.html) instead:
```suggestion
@pytest.fixture
def v1_cloudevent():
    return (
        v1.Event()
            .SetContentType("application/json")
            .SetData('{"name":"john"}')
            .SetEventID("my-id")
            .SetSource("from-galaxy-far-far-away")
            .SetEventTime("tomorrow")
            .SetEventType("cloudevent.greet.you")
    )
```

You probably want a `v03_cloudevent` fixture as well if you need it for the text that's using this below.